config/locales/icon/en.yml

```diff
@@ -1,6 +1,7 @@
 en:
   icons:
     edit: '⚙️'
+    edit_pencil: '✏️'
```

Comment:
Could this just be named `pencil`?

Reply:
renamed https://github.com/zinc-collective/convene/pull/942/commits/0a99dceb62945a62b110a009e090ae06e46c9675